src/components/explorer/IndividualTracks.js

```diff
@@ -105,19 +105,19 @@ const IndividualTracks = ({individual}) => {
             return;
         }
 
-        if (!allTracks.length || !hasFreshUrls(allTracks, igvUrls) || igvRendered.current) {
+        if (!allFoundFiles.length || !hasFreshUrls(allTracks, igvUrls) || igvRendered.current) {
```

Comment:
I don't really know about this, but is the value for `allFoundFiles` updated dynamically? It seems to me that `allTracks` will change with the component whereas `allFoundFiles` is only set at initialization(?)
Could this have an impact on this test?

Reply:
This particular test is fine, since we don't want to update `allFoundFiles` (they were found or they weren't) and the only check here is whether we found any files at all. This change addresses a bug where the igv viewer was launched even though there were no files to display.

`allTracks` _is_ mutated, but only properties of individuals tracks are changed (like whether they are viewable). In the long term I think the ideal fix is to remove the drs file searching responsibilities from this component entirely and put it in the parent component ExplorerIndividualContent,js.This will improve the overlap in responsibilities as well as eliminating some code duplication elsewhere.